### PR TITLE
Migrated TC test_no_fresh_lookup_on_directory

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_no_fresh_lookup_on_directory](https://github.com/gluster/glusto-tests/blob/master/tests/functional/disperse/test_no_fresh_lookup_on_directory.py)

Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
```
Results for disperse
Test nature : disruptive
TestName : test_no_fresh_lookup_on_directory
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|  dist-disp  |     PASS    |        00:00:35       |      NA     |
|   dist-rep  |     PASS    |        00:00:35       |      NA     |
|   dist-arb  |     PASS    |        00:00:37       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   3   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 02 minutes 02 seconds
✔ Tearing down successful.
```

**Comment by nik-redhat** [comment]
```
Results for Special
Test nature : nonDisruptive
TestName : vol_create_test
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|   dist-arb  |     PASS    |        00:00:10       |      NA     |
|   dist-rep  |     PASS    |        00:00:10       |      NA     |
|  dist-disp  |     PASS    |        00:00:08       |      NA     |
+-------------+-------------+-----------------------+-------------+
TestName : vol_destroy_test
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|   dist-arb  |     PASS    |        00:00:16       |      NA     |
|   dist-rep  |     PASS    |        00:00:16       |      NA     |
|  dist-disp  |     PASS    |        00:00:08       |      NA     |
+-------------+-------------+-----------------------+-------------+
Results for disperse
Test nature : nonDisruptive
TestName : test_no_fresh_lookup_on_directory
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|   dist-arb  |     PASS    |        00:00:47       |      NA     |
|   dist-rep  |     PASS    |        00:00:47       |      NA     |
|  dist-disp  |     PASS    |        00:00:34       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   0   |
|         Non Disruptive Tests         |   3   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   9   |
|            Tests Skipped             |   0   |
|             Total Tests              |   3   |
|   Disruptive Tests Pass Percentage   |   0   |
| Non Disruptive Tests Pass Percentage | 100.0 |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 02 minutes 20 seconds
✔ Tearing down successful.
```
Results for Special
Test nature : nonDisruptive
TestName : vol_create_test
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:07       |      NA     |
|     dist    |     PASS    |        00:00:08       |      NA     |
|     disp    |     PASS    |        00:00:08       |      NA     |
|   dist-rep  |     PASS    |        00:00:08       |      NA     |
|  dist-disp  |     PASS    |        00:00:09       |      NA     |
+-------------+-------------+-----------------------+-------------+
TestName : vol_destroy_test
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:09       |      NA     |
|     dist    |     PASS    |        00:00:08       |      NA     |
|     disp    |     PASS    |        00:00:10       |      NA     |
|   dist-rep  |     PASS    |        00:00:11       |      NA     |
|  dist-disp  |     PASS    |        00:00:09       |      NA     |
+-------------+-------------+-----------------------+-------------+
Results for fuse_subdir
Test nature : nonDisruptive
TestName : test_subdir_mount
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:29       |      NA     |
|     dist    |     PASS    |        00:00:29       |      NA     |
|     disp    |     PASS    |        00:00:35       |      NA     |
|   dist-rep  |     PASS    |        00:00:35       |      NA     |
|  dist-disp  |     PASS    |        00:00:29       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   0   |
|         Non Disruptive Tests         |   3   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   15  |
|            Tests Skipped             |   0   |
|             Total Tests              |   3   |
|   Disruptive Tests Pass Percentage   |   0   |
| Non Disruptive Tests Pass Percentage | 100.0 |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 02 minutes 44 seconds
✔ Tearing down successful.
```

File: tests/functional/disperse/test_no_fresh_lookup_on_directory.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ Description:
+    The testcase covers negative lookup of a directory in distributed-
+    replicated and distributed-dispersed volumes
+"""
+
+# nonDisruptive;dist-disp,dist-rep,dist-arb
```
Comment (Sheetalpamecha) [inline tests/functional/disperse/test_no_fresh_lookup_on_directory.py:23]: Why you converted tc from Disruptive to nonDisruptive? 
As we are making changes in cluster by bringing bricks down

**Review by Sheetalpamecha — CHANGES_REQUESTED** [review]

File: tests/functional/disperse/test_no_fresh_lookup_on_directory.py (hunk 1 above)
Comment (nik-redhat) [reply tests/functional/disperse/test_no_fresh_lookup_on_directory.py:23]: We are making changes at the volume level, not at the cluster level.

Comment (Sheetalpamecha) [reply tests/functional/disperse/test_no_fresh_lookup_on_directory.py:23]: okay

**Review by Sheetalpamecha — APPROVED** [review]

**Pull request merged**